### PR TITLE
Updated premium pennant ui icon designs to v3

### PR DESCRIPTION
**Pull request by mandla-noredink**
# :wrench: Modifying a component

## Context

This change modifies the Premium pennant icons as a part of the [Premium Vouchers project](https://paper.dropbox.com/doc/Product-Guide-Premium-Vouchers-BTS-23-G3-5-Premium-Unlock--B_2GHtbbxYDZSq0JZTydO3IIAg-u9UycOiP7B3ctBGZszNb9).

## :framed_picture: What does this change look like?

Updated premium pennants (content, active, inactive, expired):
Screenshot 2023-09-13 at 11 20 09

Gift icon for vouchered premium content:
Screenshot 2023-09-13 at 11 20 27

New icons in place:
Screenshot 2023-09-13 at 11 21 01

## Component completion checklist

- [x] I've gone through the relevant sections of the [Development Accessibility guide](https://paper.dropbox.com/doc/Accessibility-guide-4-Development--BiIVdijSaoijjOuhz3iTCJJ1Ag-rGoHpC91pFg3zTrYpvOCQ) with the changes I made to this component in mind
- [x] Changes are clearly documented
  - [x] Component docs include a changelog
  - [x] Any new exposed functions or properties have docs
- [x] Changes extend to the Component Catalog
  - [x] The Component Catalog is updated to use the newest version, if appropriate
  - [x] The Component Catalog example version number is updated, if appropriate
  - [x] Any new customizations are available from the Component Catalog
  - [x] The component example still has:
    - an accurate preview
    - valid sample code
    - correct keyboard behavior
    - correct and comprehensive guidance around how to use the component
- [ ] Changes to the component are tested/the new version of the component is tested
- [x] Component API follows standard patterns in noredink-ui
  - e.g., as a dev, I can conveniently add an `nriDescription`
  - and adding a new feature to the component will _not_ require major API changes to the component
- [x] If this is a new major version of the component, our team has stories created to upgrade all instances of the old component. Here are links to the stories:
  - [Change Premium Flag icons in their 4 states](https://linear.app/noredink/issue/GRO-1388/change-premium-flag-icons-in-their-4-states)
  - [Add gift icons for G3-5 premium content](https://linear.app/noredink/issue/GRO-1389/add-gift-icons-for-g3-5-premium-content)
- [x] Please assign the following reviewers (applicable Sep 2023–Dec 2023):
  - [x] [a11y-volunteer-reviewers](https://github.com/orgs/NoRedInk/teams/volunteer-a11y-reviewers) - Someone from this group will review your PR in full.
  - [x]  [team-accessibilibats-a11ybats](https://github.com/orgs/NoRedInk/teams/team-accessibilibats-a11ybats) - Someone from this group will review your PR for ACCESSIBILITY ONLY.
  - [x]  Someone from your team who can review requirements from your team's perspective. (This could be the same person from the [a11y-volunteer-reviewers](https://github.com/orgs/NoRedInk/teams/volunteer-a11y-reviewers) group if you'd like.)
  - [x]  If there are user-facing changes, a designer:
    - For writing-related component changes, add Stacey (staceyadams)
    - For quiz engine-related components, add Ravi (ravi-morbia)
    - For a11y-related changes to general components, add Ben (bendansby)
    - For general component-related changes or if you’re not sure about something, add the Design group (NoRedInk/design)

### CODE REVIEW

File: src/Nri/Ui/Pennant/V3.elm [hunk 1]
```diff
@@ -0,0 +1,161 @@
+module Nri.Ui.Pennant.V3 exposing (contentPremiumFlag, activePremiumFlag, inactivePremiumFlag, expiredPremiumFlag, giftPremiumFlag)
+
+{-| Changes from V3:
```
Comment (charbelrami) [inline src/Nri/Ui/Pennant/V3.elm:3]:
```suggestion
{-| Changes from V2:
```

**Review by charbelrami — COMMENTED** [review]
There is an issue with the svg dimensions:

new version

They should have the same size and occupy most of the viewbox like these:

old version

--- 

The PR looks good from an accessibility perspective!

File: src/Nri/Ui/Pennant/V3.elm (hunk 1 above)
Comment (mandla-noredink) [reply src/Nri/Ui/Pennant/V3.elm:3]: Regarding the `contentPremiumFlag`, the icon is supposed to be smaller by default. Is that a difference we can keep if I update the docs for the component to mention this explicitly?

**Comment by mandla-noredink** [comment]
> There is an issue with the svg dimensions:
> 
> new version
> They should have the same size like these:
> 
> old version
> The PR looks good from an accessibility perspective!

Regarding the contentPremiumFlag, the icon is supposed to be smaller by default. Is that a difference we can keep if I update the docs for the component to mention this explicitly?

I've updated the gift icon to be the same size as the bigger pennants, but kept them less than the full height of the icon window to preserve the size ratio with the smaller pennant by default. Does that work?

**Review by charbelrami — APPROVED** [review]
Thanks! Looks good!

**Pull request merged**